src/network/networkd-dhcp-server.c

```diff
@@ -45,7 +45,7 @@ static int link_push_uplink_dns_to_dhcp_server(Link *link, sd_dhcp_server *s) {
         size_t n_addresses = 0, n_allocated = 0;
         unsigned i;
 
-        log_link_debug(link, "Copying DNS server information from %s", link->ifname);
+        log_link_debug(link, "Copying DNS server information");
```

Comment:
Hmm, this will be rather misleading. When I see "hub0: Copying DNS server information from hub0", the meaning is pretty easy. But without the last two words, I would assume that the copy happens in the other direction. So maybe reword the message somehow, e.g. "Copying DNS server information from link"?

Reply:
Updated the messages as suggested.